### PR TITLE
Add new function to read text file

### PR DESCRIPTION
**Pull request by Abdullamhd**

### CODE REVIEW

File: draft-data/main.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+
+def read_text(file_name):
```
Comment (MLVisionPlatform) [inline draft-data/main.py:3]: this function should be changed to enhance performance

**Review by MLVisionPlatform — CHANGES_REQUESTED** [review]

File: draft-data/main.py (hunk 1 above)
Comment (Abdullamhd) [reply draft-data/main.py:3]: the issue has been addressed

File: draft-data/main.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    # this will open the file and read it into a string
```
Comment (Abdullamhd) [inline draft-data/main.py:4]: this is the comment i leave it

**Review by MLVisionPlatform — APPROVED** [review]
the fix has been approved

**Pull request merged**